### PR TITLE
Guides: link to discourse

### PR DESCRIPTION
**Pull request by andesol**
## Done 

Instead of having a dedicated guides page, let's just link to discourse. The rationale for this can be found [in the spec](https://docs.google.com/document/d/1RI5N2rbasGZid1Vr2CdMhIKaEirT06PjU-bgjnENTn8/edit):

> ### The structure of the Guides
> Currently the Guides page uses a similar structure to other docs pages - it has a manually maintained side navigation, with grouping and nesting.
> As the guides are much less structured than our other docs or Practices site, it’s sometimes difficult to place a given guide in just one category, or find a category for it at all. It may be much more useful to be able to search those, or group them by keywords or tags (“docker” “k8s” “demos” “assets”).
> Various ideas to consider:
> - Adding search to the guides
> - Automatically generating list of guides from discourse API (initially proposed by [Goulin Khoge](mailto:goulin.khoge@canonical.com))
> - Showing a sortable/filterable list of guides instead of page with side navigation
> Although many of those seem to mean reimplementing some of native discourse features. So maybe it would be useful to keep Guides *just in the discourse itself*, where they can be searched for, tagged, etc.

## QA

- Open demo
- Click on Guides in top navigation
- Check it goes to the guides section on discourse 

## Issue

Fixes https://github.com/canonical/Web-design-systems-squad/issues/54

### CODE REVIEW

**Comment by bartaz** [comment]
It is being discussed in the spec, not sure if we decided it's the best way to do it? @nottrobin

**Comment by bartaz** [comment]
Now, that the guides are not part of the site, do we keep them in the main nav?

image

It's a bit weird to keep it there - it now being an external link to separate website, and hidden behind SSO.

But if not there, where to put it? Maybe on the home page of the portal?

**Comment by andesol** [comment]
It certainly removes some duplication and having to keep the navigation updated. Plus it may drive more people to our discourse (less clicks to add comments and do changes). Since the content of the site is mostly for internal usage, I think it's ok to keep it in the navigation. 

But it has its downsides too, for sure. Mostly opened this to have a working demo. Let's ask anyone who has an opinion on this.

File: templates/base.html [hunk 1]
```diff
@@ -1,83 +1,135 @@
...
+                  />
+                </div>
+                <span class="p-navigation__logo-title">Canonical</span>
+              </a>
+            </div>
+            <a
+              href="#navigation"
+              class="p-navigation__toggle--open"
+              title="menu"
+              >Menu</a
```
Comment (carkod) [inline templates/base.html:41]: I think Prettier doesn't format HTML very well, it tends to break it html closing tags incorrectly.

File: templates/base.html [hunk 2]
```diff
@@ -1,83 +1,135 @@
...
-              <li class="p-inline-list__item">
-                <a href="https://github.com/canonical-web-and-design/webteam.canonical.com/issues/new">Report a bug with this site</a>
+            </ul>
+            <ul class="p-navigation__items" role="menu">
+              <li class="p-navigation__item">
+                <a
+                  class="p-navigation__link"
+                  href="https://github.com/canonical-web-and-design"
+                  ><i class="p-icon--github"></i
+                ></a>
```
Comment (carkod) [inline templates/base.html:86]: Here too, it's probably better to `</i>`?

**Comment by minkyngkm** [comment]
@andesol Could you take a look at this PR?

File: templates/base.html (hunk 1 above)
Comment (andesol) [reply templates/base.html]: Thanks! I reverted the formatting.

**Comment by andesol** [comment]
@minkyngkm Formatting changes done. I still need an approval, though. cc/ @bartaz @nottrobin

**Comment by nottrobin** [comment]
> @bartaz: It is being discussed in the spec, not sure if we decided it's the best way to do it? @nottrobin

I think this is the neatest solution, we should just go for it.

> @bartaz: Now, that the guides are not part of the site, do we keep them in the main nav?
> It's a bit weird to keep it there - it now being an external link to separate website, and hidden behind SSO.
> But if not there, where to put it? Maybe on the home page of the portal?

> @andesol: It certainly removes some duplication and having to keep the navigation updated. Plus it may drive more people to our discourse (less clicks to add comments and do changes). Since the content of the site is mostly for internal usage, I think it's ok to keep the link in the top navigation.

It was already behind SSO, so that hasn't changed. The only difference really is that it's on a different domain with different styling.

What is the difference between the top nav and the main links on the webteam.canonical.com homepage anyway? Surely it's only prominence - we're putting the most important things up top. I think guides continues to be important, so I'd probably keep it in the top nav, personally.

So overall I think this PR is great as-is, I'd like to merge it. @anthonydillon if you happen to see this maybe you have an opinion? Otherwise let's go for it.

**Review by nottrobin — APPROVED** [review]

**Comment by anthonydillon** [comment]
I guess this is ok for internal use only. The discourse content did allow us to define a navigation which we would now lose.

Maybe we should label it as such for better UX.

**Comment by nottrobin** [comment]
> I guess this is ok for internal use only. The discourse content did allow us to define a navigation which we would now lose.

Losing the navigation is a design decision we're making here - if you read the description of the PR. Guides are so many and various and always being added, which makes it very hard to keep the navigation up to date. It also adds a burden to anyone writing a guide, which may put them off doing so. I think a search is a fundamentally better way to navigate guides.

> Maybe we should label it as such for better UX.

👍. Maybe a padlock icon? 🔒

**Pull request merged**